init.lua

```diff
@@ -528,7 +528,7 @@ minetest.register_on_joinplayer(function(player)
 	local level = stamina.get_saturation(player) or settings.visual_max
 	local id = player:hud_add({
 		name = "stamina",
-		hud_elem_type = "statbar",
+		type = "statbar",
```

Comment:
```suggestion
		[minetest.features.hud_def_type_field and "type" or "hud_elem_type"] = "statbar",
```
Required to keep compatibility w/ Luanti 5.8.0 (server) and older. See https://github.com/luanti-org/luanti/blob/0eb047ca33b2c3708257d95f8857697fc0403c07/doc/lua_api.md?plain=1#L5651-L5652